packages/admin-theme/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@comet/admin-theme",
+    "version": "1.0.0",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "files": [
+        "lib"
+    ],
+    "dependencies": {},
+    "devDependencies": {
+        "@comet/admin": "^1.3.0",
+        "@comet/admin-rte": "^1.2.1",
+        "@comet/admin-color-picker": "^1.0.2",
+        "@comet/admin-react-select": "^1.0.2"
```

Comment:
those have to be peerDependencies

Reply:
@nsams are you sure, `peerDependencies` is correct?  
When i do this, build and pipeline fail, because `@comet/admin` cannot be found in `packages/admin-color-picker/src/core/ColorPicker.tsx:1` for some reason...

When I add them to `devDependencies` it works.  

The only reason, the `@comet/*`  dependencies are needed, is for importing the types `CometAdmin...ClassKeys`.

Reply:
@nsams the way I understand `peerDependencies`, they **must** be installed by the "parent" (Admin-Project, using the theme). 

In this case, they should not be required.  

The Admin-Project might not include a color-picker and therefore sould not be required to install `@comet/admin-color-picker`. `@comet/admin-theme` only uses types from the color-picker. As far as I know, this is only required during development and build. Wouldn't `devDependencies` be the correct way of installing these packages then?

Reply:
except for `@comet/admin` as you use createMuiTheme

Reply:
`createMuiTheme` is now done outside the theme-package, because of 779cd7f3e759dfb247bab1b37ea657413f28c984.